live-examples/js-examples/set-prototype-entries.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">var set1 = new Set();
+set1.add(42);
+set1.add('forty two');
+
+var iterator1 = set1.entries();
+
+console.log(iterator1.next().value);
+// expected output: [42, 42]
+
+console.log(iterator1.next().value);
+// expected output: ["forty two", "forty two"]
+</code>
```

Comment:
what a mad API!

Reply:
This is strictly optional of course, but using `for...of` seems more like a normal use than `iterator.next().value`:

```
var set1 = new Set();
set1.add(42);
set1.add('forty two');

var iterator1 = set1.entries();

for (let entry of iterator1) {
  console.log(entry);
}
```

Reply:
ok

Reply:
I suppose the advantage of using `iterator.next().value` is that it makes the `console.log` output slightly clearer, maybe. Bur `for...of` is neater, so swings and roundabouts :)